### PR TITLE
Done modifications

### PR DESCRIPTION
**Pull request by Rohithgilla12**

### CODE REVIEW

**Comment by Rohithgilla12** [comment]
Btw I have updated the readme file and added badeges should i pr that?

File: xkcd.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import re
+import requests
+from bs4 import BeautifulSoup
+import random
+import os
+
+
+l=[] #For storing random numbers
```
Comment (srbcheema1) [inline xkcd.py]: Why to declare it so earlier ? it should be near the code it is being used in.
Actually I am not gonna use it in the code(read my next comment for followup). Just telling to declare variables close to the code block they are used in, unless they are used at many places.

File: xkcd.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    else:
+        os.mkdir('XKCD') #If false create a folder
+    op_file=open('XKCD/'+filename,'wb')
+    op_file.write(data) #Download off
+    print "Downloaded",filename
+
+
+choice=input("How many random images you want to download?? xD \n")
+
+for i in range(choice):
```
Comment (srbcheema1) [inline xkcd.py]: there is no need of this loop and no need of `l=[]`
you could simply generate a random number on the go.

File: xkcd.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    l.append(str(random.randint(1,1933))) #Last comic till date is 1933
+
+for i in l:
```
Comment (srbcheema1) [inline xkcd.py]: do this as
```
for i in range(choice):
   num = str(random.randint(1,1933))
   url = "http://xkcd.com/"+num+"/"
```

File: README.md [hunk 4]
```diff
@@ -3,6 +3,10 @@
 
 ![python logo](https://www.python.org/static/community_logos/python-powered-w-200x80.png)
 
+
+[![forthebadge](https://forthebadge.com/images/badges/built-with-love.svg)](https://forthebadge.com) [![forthebadge](https://forthebadge.com/images/badges/made-with-python.svg)](https://forthebadge.com)
+
+
```
Comment (srbcheema1) [inline README.md:9]: thanks for this :)
it looks great

File: CricketNotifications.py [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+n.set_urgency(notify2.URGENCY_NORMAL)
+n.set_timeout(1000)
+url="http://synd.cricbuzz.com/j2me/1.0/livematches.xml"
+r=requests.get(url)
+soup=BeautifulSoup(r.content,'html.parser')
+temp=soup.find_all('match')
+series_names=[]
+datapath=[]
+for i in temp:
+
```
Comment (srbcheema1) [inline CricketNotifications.py]: remove this empty line

**Review by srbcheema1 — CHANGES_REQUESTED** [review]

**Comment by srbcheema1** [comment]
I have requested small changes.
After this I will merge this PR
also thanks for the README update.
it really looks nice :smiley:

**Comment by Rohithgilla12** [comment]
Done modifications :D

**Comment by srbcheema1** [comment]
looks good to me now :relaxed:

**Review by srbcheema1 — APPROVED** [review]
just one last thing before we merge this PR.you may sqash all those commits into one commit. this is optional and its your wish. you will learn a cool thing that is highly useful in open source. I am happy with the PR. just you may do it if you want else . tell me if you are going to squash them into one. otherwise its fine . i will be waiting to merge it :)

**Comment by Rohithgilla12** [comment]
Hey how to squash all commits into one, I tried but i didn't understand the way.
Can you help me in that please

**Comment by Rohithgilla12** [comment]
Yeah, Dude merge it now :)

**Comment by srbcheema1** [comment]
Merged :)
You may now have a look at https://github.com/srbcheema1/CheemaFy
this is my favourite repository.
If you use ubuntu you will surely like it

**Comment by Rohithgilla12** [comment]
I use a Mac.
I'll see it and try to contribute to it :)

**Comment by srbcheema1** [comment]
I have recently added support for mac. but thats in alpha state. i m working on that too.

**Pull request merged**